var/spack/repos/builtin/packages/py-pillow/package.py

```diff
@@ -25,6 +25,7 @@ class PyPillowBase(PythonPackage):
     variant("jpeg2000", default=False, description="JPEG 2000 functionality")
     variant("imagequant", when="@3.3:", default=False, description="Improved color quantization")
     variant("xcb", when="@7.1:", default=False, description="X11 screengrab support")
+    variant("raqm", when="@8.2:", default=False, description="RAQM support")
```

Comment:
Tested this with in-between versions and it looks like 8.2.0 is the first version to support this flag.